community/people/CGK ; )/index.md

```diff
@@ -0,0 +1,20 @@
+---
+Layout: Person
+Title: Carena
+Role: Summer Intern
+Image: Go to Picture.jpeg
+Email: cgkelly98@gmail.com
+Room: Virtual (Atlanta, Ga)
+---
+
+
+## Research Interests
+
+* Nuclear nonproliferation
+* Nuclear Security
+* All things Nuclear (Undecided)
+
+
+## Education
+
+* South Carolina State University Senior
```

Comment:
I think this whole file has been added by mistake, it is a copy of `community/people/CGK/index.md` but with some strange path...